src/components/VideoPlayer/index.js

```diff
@@ -171,6 +171,78 @@ export default class VideoPlayer extends PureComponent {
 
   resumeVideo = () => { this.video.play() }
 
+  handleKeyDown = (e) => {
```

Comment:
if you wanted to, you could pull `target` and `preventDefault` here since that's all that's used.  not required, just a suggestion

Reply:
that's cool

Reply:
looks like destructuring `preventDefault` out doesn't work unless I bind it to e again